package.json

```diff
@@ -29,5 +29,11 @@
     "lerna": "^3.20.2",
     "prettier": "^2.0.5",
     "typescript": "^3.8.3"
+  },
+  "dependencies": {
+    "@transmute/did-key-ed25519": "^0.2.1-unstable.24",
+    "@transmute/did-key-x25519": "^0.2.1-unstable.24",
+    "@w3c-ccg/universal-wallet": "file:packages/universal-wallet",
```

Comment:
```suggestion
    "@transmute/universal-wallet": "^0.4.1",
```

Reply:
This is required to fix lerna issues in this PR.

Reply:
lerna automatically handles linking, so the `file:` is not needed, and breaks things.

Reply:
updated in commit 733b20921295b8f495ff0ad1776a9aeb698b521b